.github/workflows/cd.yml

```diff
@@ -172,4 +189,30 @@ jobs:
         with:
           body_path: CHANGELOG.md
           files: |
-            ./artifacts/*.zip
\ No newline at end of file
+            ./artifacts/*.zip
+
+  bump-formulae:
+    needs: github-release
```

Comment:
👍 yup, this seems right to me. It ... looks like it'll "just work"?

I think some other repos i've found use [this bump action](https://github.com/mislav/bump-homebrew-formula-action). Could try that one if the Homebrew/ one doesn't work.

Reply:
I hope so, wish there was a way to make sure before making a release.
This other action looks good as well

Reply:
Would you say it's worth changing to the non official one?

Reply:
honestly no idea. I think either of them should work, but maybe more people use the non-official one? I would say keep as-is and see what happens on the next tagged release.

Reply:
Alright